modules/machineset-azure-accelerated-networking.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-azure.adoc
+
+[id="machineset-azure-accelerated-networking_{context}"]
+= Accelerated Networking for Microsoft Azure VMs
+
+Accelerated Networking uses single root I/O virtualization (SR-IOV) to provide Microsoft Azure VMs with a more direct path to the switch. This results in enhanced network performance. This feature can be enabled during or after installation.
+
+[id="machineset-azure-accelerated-networking-limits_{context}"]
+== Limitations
+
+Consider the following limitations when deciding whether to use Accelerated Networking:
+
+* Accelerated Networking is supported on clusters with installer-provisioned infrastructure only.
+
+* By default, {product-title} worker nodes are dual core. Accelerated Networking requires an Azure VM size that includes at least four vCPUs. To satisfy this requirement, you can change the value of `vmSize` in your machine set. For information about Azure VM sizes, refer to link:https://docs.microsoft.com/en-us/azure/virtual-machines/sizes[Microsoft Azure documentation].
+
+* When this feature is enabled on an existing Azure cluster, only newly provisioned nodes are affected. Currently running nodes are not reconciled. To enable the feature on all nodes, you must scale down replicas to zero, and then scale back up to your desired number of replicas.
```

Comment:
Scaling down to zero and back up again probably isn't something most users will want to do on a production cluster, perhaps we can just say replace each existing machine?

Reply:
That makes sense - I assume that's more of a one-at-a-time approach, giving them a bit more control of what to take down when? How do you feel about leaving the scale down and up language in (for, let's say, someone coming in after install but before things are really in use), but also add the suggestion of replacing each (for an actual established prod cluster)?

Reply:
SGTM